clients/java/src/test/java/io/unitycatalog/client/EnumDeserializationTest.java

```diff
@@ -23,18 +23,16 @@
 import org.junit.jupiter.api.Test;
```

Comment:
No code change,  automatically reformatted after applied the `build/sbt client/javafmtAll`.